src/ECER.Clients.RegistryPortal/ecer.clients.registryportal.client/src/components/FileUploader.vue

```diff
@@ -64,16 +87,53 @@ export default defineComponent({
       selectedFiles: [],
       showErrorBanner: false,
       errorBannerMessage: "",
+      firstLoad: true,
```

Comment:
Would it work to remove `firstLoad` and the watcher on `selectedFiles` and instead jut emit when we either add/remove files from `selectedFiles `list? I just find with watchers we're always dealing with the unintended side effects, like what you having to fix with the `firstLoad ` flag. Thoughts?

Reply:
discussed to be done in a future refactor ticket. 

Worried about scope of this on this ticket since it will affect how replyToMessage works.